Tests/UnitTests/Purchasing/ReceiptStringsTests.swift

```diff
@@ -212,16 +212,6 @@ final class ReceiptStringsTests: TestCase {
         expect(subject.description).to(equal(expectedDescription))
     }
 
-    func testReceiptRetryingMechanismNotAvailable() {
-        let subject = ReceiptStrings
-            .receipt_retrying_mechanism_not_available
-        let expectedDescription = "Receipt retrying mechanism is not available in iOS 12. " +
-        "Will only attempt to fetch once."
-
-        expect(subject.category).to(equal("receipt"))
-        expect(subject.description).to(equal(expectedDescription))
-    }
```

Comment:
Looks like this was a thing in iOS 12, and we've dropped support for that already. See b3875a9

Reply:
Yep. In fact it seems it was removed v5.0.0 (https://github.com/RevenueCat/purchases-ios/commit/2bcd0ce3287c391a292825228666d3147074b16c#diff-b90ac6f010c07896e177bbd44986b42d86117183a4cc0af00417ab0a00d8c361)